### PR TITLE
Dg starting on UI

### PR DESCRIPTION
**Pull request by dgarcia13-99**
Added navigation bar which contains resources, or links, to the index page for 'Photos', 'Comments', 'Follow_Requests', and 'Likes'. Used Devise methods to allow users to create an account or sign in to view Photogram's resources.

### CODE REVIEW

**Review by hatcheta1 — APPROVED** [review]
looks great! nice use of partials

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb]: nit pick remove the space here

**Review by aldoportillo — CHANGES_REQUESTED** [review]

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (dgarcia13-99) [reply app/views/shared/_navbar.html.erb]: just fixed it! Thanks for the feedback

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb]: good job Dani

**Review by aldoportillo — APPROVED** [review]

**Pull request merged**